src/middleware/index.js

```diff
@@ -183,6 +186,20 @@ export let setupSocketIOAuthentication = function(app, options = {}) {
         }).catch(errorHandler);
       }
     });
+
+    socket.on('logout', function() {
```

Comment:
Ha! I had started doing it that way but then I switched to be consistent with the `authenticate` and `authenticated` events. I think callbacks is probably a better idea.